src/main/java/org/mskcc/limsrest/controller/sequencingqc/UpdateLimsSampleLevelSequencingQc.java

```diff
@@ -0,0 +1,47 @@
+package org.mskcc.limsrest.controller.sequencingqc;
+
+import com.velox.sloan.cmo.recmodels.SeqAnalysisSampleQCModel;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.mskcc.limsrest.ConnectionLIMS;
+import org.mskcc.limsrest.service.sequencingqc.UpdateLimsSampleLevelSequencingQcTask;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Map;
+
+/**
+ * Endpoint to Add/Update Sequencing QC Stats to LIMS SeqAnalysisSampleQC DataType. The endpoint is planned to run as
+ * soon as new Sequencing Stats are posted to NGS-STATS database.
+ */
+@RestController
+@RequestMapping("/")
+public class UpdateLimsSampleLevelSequencingQc {
+    private static Log log = LogFactory.getLog(UpdateLimsSampleLevelSequencingQc.class);
+    private final ConnectionLIMS conn;
+
+    public UpdateLimsSampleLevelSequencingQc(ConnectionLIMS conn){
+        this.conn = conn;
+    }
+
+    @GetMapping("/updateLimsSampleLevelSequencingQc")
+    public Map<String, String> getContent(@RequestParam(value = "runId") String runId) {
+        if (StringUtils.isBlank(runId)){
+            log.info(String.format("Invalid RUN ID: '%s'", runId));
+            return null;
```

Comment:
Can you return an error message? Something like,

```
final String error = String.format("Invalid RUN ID: '%s'", runId);
Map resp = new HasnMap<String, String>();
resp.put("error", error);
return resp
```

In nextflow, it's possible to add logic that will let us know whether or not a part of the workflow failed. So we could check to see if curling `/updateLimsSampleLevelSequencingQc` returns an error to fail that process.